pool/pool.go

```diff
@@ -1582,3 +1582,30 @@ func newAddressFromCnrID(cnrID *cid.ID) *address.Address {
 	addr.SetContainerID(cnrID)
 	return addr
 }
+
+func copySessionTokenWithoutSignature(from session.Token) (to session.Token) {
```

Comment:
Actually, the context is also not copied

Reply:
Updated function name and test.